Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -395,6 +395,42 @@ private SKBitmap OrientImage(SKBitmap bitmap, SKEncodedOrigin origin)
             return rotated;
         }
 
+        /// <summary>
+        /// Resizes an image on the CPU, by utilizing a surface and canvas.
+        /// </summary>
+        /// <param name="source">The source bitmap.</param>
+        /// <param name="targetInfo">This specifies the target size and other information required to create the surface.</param>
+        /// <param name="isAntialias">This enables anti-aliasing on the SKPaint instance.</param>
+        /// <param name="isDither">This enables dithering on the SKPaint instance.</param>
+        /// <returns>The resized image.</returns>
+        internal static SKImage ResizeImage(SKBitmap source, SKImageInfo targetInfo, bool isAntialias = false, bool isDither = false)
+        {
+            using var surface = SKSurface.Create(targetInfo);
+            using var canvas = surface.Canvas;
+            using var paint = new SKPaint();
+
+            paint.FilterQuality = SKFilterQuality.High;
+            paint.IsAntialias = isAntialias;
+            paint.IsDither = isDither;
+
+            var kernel = new float[9]
+            {
+                        0,    -.1f,    0,
+                        -.1f, 1.4f, -.1f,
+                        0,    -.1f,    0,
+            };
```

Comment:
What do these numbers mean?

```suggestion
            var kernel = new float[9]
            {
                0,    -.1f,    0,
                -.1f, 1.4f, -.1f,
                0,    -.1f,    0,
            };
```

Reply:
These are the coefficients for a convolutional filter for slight sharpening during rendering: https://en.wikipedia.org/wiki/Kernel_(image_processing)#Details